celements-model/src/main/java/com/celements/model/access/DefaultXWikiDocumentCreator.java

```diff
@@ -20,19 +22,22 @@
 
   @Requirement
   private ModelContext context;
+  @Requirement
+  private ModelUtils modelUtils;
 
   @Override
   public XWikiDocument createWithoutDefaults(DocumentReference docRef, String lang) {
     XWikiDocument doc = new XWikiDocument(cloneRef(docRef, DocumentReference.class));
     doc.setNew(true);
+    lang = modelUtils.normalizeLang(lang);
```

Comment:
We must additionally check against `getDefaultLangForCreatingDoc(docRef, lang)`. If the defaultLang.equals(lang) we must set it to `DEFAULT_LANG`.